rails-commenting.rb

```diff
@@ -39,7 +39,7 @@ def edit
 
   def update
     @post = BlogPost.find(params[:id])
-    # 6)
+    # 6)its used to update an entry
```

Comment:
It is used  to update an entry using blog_post_params to only allow proper content